JSDemos/Demos/PivotGrid/LayoutCustomization/description.md

```diff
@@ -1 +1,12 @@
-The PivotGrid component supports multiple customizations allowing you to&nbsp;provide the look and feel that will satisfy any user. This demo illustrates only a&nbsp;small part of&nbsp;the available customizations. Toggle the check boxes above the PivotGrid to&nbsp;change the layout at&nbsp;runtime.
\ No newline at end of file
+PivotGrid allows you to customize its layout. This demo illustrates the following properties:
+
+- [rowHeaderLayout](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#rowHeaderLayout)            
+Allows you to specify layout type for the row header items. The *"standard"* layout displays child items on the right from the parent items. Assign "*tree*" to this property to display items more compactly as a hierarchical structure. 
```

Comment:
- "on the right from" - [68 600 000 results](https://www.google.ru/search?q=%22on+the+right+from%22&newwindow=1&bih=927&biw=1920&hl=ru&sxsrf=AOaemvIN8JB9J0uR6Pfich3INvEtRruglg%3A1631882948907&ei=xI5EYZXZNv6SwPAPsKev4Ao&oq=%22on+the+right+from%22&gs_lcp=Cgdnd3Mtd2l6EAM6BwgjELADECc6BwgAEEcQsAM6BwgjEOoCECc6BAgjECc6CwgAEIAEELEDEIMBOggIABCABBCxAzoOCC4QgAQQsQMQxwEQowI6EQguEIAEELEDEIMBEMcBENEDOgsILhCABBDHARCjAjoRCC4QgAQQsQMQxwEQowIQkwI6BQgAEIAEOggIABCxAxCDAToFCC4QgAQ6CAguEIAEELEDOgsILhCABBCxAxCTAjoICC4QgAQQkwI6BQgAEMsBOgUILhDLAToECAAQQzoHCAAQsQMQQzoOCC4QgAQQsQMQxwEQ0QM6EAguEIAEEIcCELEDEIMBEBQ6CggAEIAEEEYQ_wE6BggAEBYQHjoICAAQFhAKEB5KBAhBGABQkcYOWKaRD2DymQ9oEnACeACAAa0BiAGqGpIBBTI0LjExmAEAoAEBsAEKyAEJwAEB&sclient=gws-wiz&ved=0ahUKEwiVy-_bhYbzAhV-CRAIHbDTC6wQ4dUDCA0&uact=5)
- "to the right of" - [246 000 000 results](https://www.google.ru/search?q=%22to+the+right+of%22&newwindow=1&bih=927&biw=1920&hl=ru&sxsrf=AOaemvICrnR824G3GYl7l2_xitB_z9Saww%3A1631883198805&ei=vo9EYbXMMKWEwPAP9KScgAY&oq=%22to+the+right+of%22&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEAcQHjIFCAAQywEyBQgAEMsBMgUIABDLATIFCAAQywEyBQgAEMsBMgUIABDLATIFCAAQywEyBQgAEMsBMgUIABDLAToJCAAQsAMQBxAeOgkIABCwAxAIEB46BAgAEEM6BAguEEM6CwgAEIAEELEDEIMBOggILhCABBCxAzoFCAAQgAQ6BQguEIAEOggIABAHEAoQHjoHCC4QQxCTAjoFCC4QywFKBAhBGAFQ35gGWNikBmDsswZoAXAAeACAAZcBiAH_DJIBBDMuMTKYAQCgAQHIAQLAAQE&sclient=gws-wiz&ved=0ahUKEwi1o4TThobzAhUlAhAIHXQSB2AQ4dUDCA0&uact=5)

The second variant is overwhelmingly more popular. I would have used it instead.

Note that you specify the one layout type for [all] header items in general.
```suggestion
Allows you to specify the layout type for row header items. The *"standard"* layout displays child items to the right of the parent items. Assign "*tree*" to this property to display items more densely as a hierarchical structure. 
```

P.S.
> Assign "*tree*" to this property to display items more densely **as a hierarchical structure**.

What does this addition tells us in this context? One thing is to force your items to occupy as less space as possible, while the other is to arrange the items into a hierarchy. What does this property do? Do the items occupy less space because they are hierarchically arranged or were they initially arranged into a hierarchy? This is not clear from your description.

Reply:
Surely the items are initially arranged into a hierarchy. All properties that we describe here relate only to visual representation of data and means of it organization. Let me show you the effect of this property: 
![image](https://user-images.githubusercontent.com/33040934/133969107-3331b801-ee81-4f4c-bcbf-cf8ae7cdf51c.png)
![image](https://user-images.githubusercontent.com/33040934/133969116-4a88a75e-df06-4cbb-afa8-8d4bc372fdfd.png)

Reply:
Alright, I see. Since this is a demo description, I believe we don't need to go into greater details in text. However, it's hard to figure out without visuals.

Reply:
Agree that it's hard to get it from text. But users can try the demo to see what happens, that's the point.